packages/react/src/components/ComposedModal/ComposedModal.js

```diff
@@ -91,7 +91,7 @@ export default class ComposedModal extends Component {
     /**
      * Specify the CSS selectors that match the floating menus
      */
-    selectorsFloatingMenus: PropTypes.string,
+    selectorsFloatingMenus: PropTypes.arrayOf(PropTypes.string),
```

Comment:
I think this is technically a breaking change, would we need to do `PropTypes.oneOfType()` and then support either option until the next major release?

Reply:
I would suggest not, on the following basis: the people is currently declared as a string, BUT actually passing a string causes a JavaScript error that aborts rendering. So there is no utility at all in actually passing a string, so supporting it as an alternative option isn't useful in this case.

Reply:
Nice, makes sense to me! Thanks so much for clarifying.